packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -149,7 +148,16 @@ export const Headline = styled.h3`
 
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     ${({ script, topStory }) =>
-      script && (topStory ? getParagon(script) : getGreatPrimer(script))}
+      script &&
+      (topStory
+        ? css`
+            font-size: ${script.paragon.groupD.fontSize / 16}rem;
```

Comment:
Howcome this is needed? Do the `getGreatPrimer` and `getParagon` functions not work as expected? Or are we trying to use 1 group over 2 breakpoints?

Reply:
It's needed for IE11 - because it wasn't picking up on the font-size and line-height specified here https://github.com/bbc/psammead/blob/f45a59d02dcd9f9120da9d9b6101dc7f2cdf37c2/packages/utilities/gel-foundations/src/getTypeSizes.js#L30 since what is returned by either getGreatPrimer() or getParagon() will have a nested media-query

Reply:
Ohhhh hmmm thats annoying of IE :P

Reply:
Ohhhh hmmm thats annoying of IE :P Any opinions @sareh @pjlee11 @jblaut? This PR works, but if this is gunna keep coming up, we may need to reevaluate our font size logic

Reply:
Is it only nested media queries that IE11 doesn't support? I'm interested in how we currently do this on the PAL, maybe we only have a single level of media queries ...

Reply:
It feels unique to story promo, where it switches typography at different breakpoints, which is  very uncommon, maybe were ok with it being in this 1 place?